src/Lykke.Job.TransactionHandler/Queues/TradeQueue.cs

```diff
@@ -16,83 +17,89 @@
 
 namespace Lykke.Job.TransactionHandler.Queues
 {
-    public sealed class TradeQueue : IQueueSubscriber
+    public sealed class TradeQueue : IStartable, IDisposable
     {
         private const bool QueueDurable = true;
 
         private readonly ILog _log;
+        private readonly string _mongoConnectionString;
```

Comment:
I would prefer to use RabbitMqSettings and MongoDeduplicator instead of lot of strings.